src/parser.rs

```diff
@@ -20,25 +21,42 @@ use crate::ImportDef;
 use crate::Location;
 use crate::ReexportKind;
 
+use deno_ast::swc::ast::ClassDecl;
 use deno_ast::swc::ast::Decl;
 use deno_ast::swc::ast::DefaultDecl;
+use deno_ast::swc::ast::ExportDecl;
+use deno_ast::swc::ast::ExportDefaultDecl;
+use deno_ast::swc::ast::ExportDefaultExpr;
 use deno_ast::swc::ast::ExportSpecifier;
-use deno_ast::swc::ast::Expr;
+use deno_ast::swc::ast::FnDecl;
+use deno_ast::swc::ast::Ident;
 use deno_ast::swc::ast::ImportSpecifier;
 use deno_ast::swc::ast::ModuleDecl;
 use deno_ast::swc::ast::ModuleItem;
-use deno_ast::swc::ast::Stmt;
+use deno_ast::swc::ast::TsEnumDecl;
+use deno_ast::swc::ast::TsInterfaceDecl;
+use deno_ast::swc::ast::TsModuleDecl;
+use deno_ast::swc::ast::TsModuleName;
+use deno_ast::swc::ast::TsTypeAliasDecl;
+use deno_ast::swc::ast::VarDecl;
 use deno_ast::swc::ast::VarDeclKind;
+use deno_ast::swc::ast::VarDeclarator;
 use deno_ast::ParsedSource;
+use deno_ast::SourceRange;
```

Comment:
So many of these... Maybe we should qualify them as `ast::FnDecl`, etc? Can be done in a follow up as it's a mechanical refactor